services/fuzzing-decision/src/fuzzing_decision/decision/pool.py

```diff
@@ -532,7 +536,7 @@ def build_resources(
         if self.cloud != "static":
             yield WorkerPool(
                 config=config,
-                description=DESCRIPTION.replace("\n", "\\n"),
```

Comment:
Did this change?

Reply:
Actually I just went to grab an example for you and I think this is wrong. There are 11 places we use `DESCRIPTION` in this file .. some escape `\n` and some don't. I know I've seen some in the TC UI that are wrong.

I'll revert this and revisit later. It's not important.